lib/filewatch/yielding_tail.rb

```diff
@@ -48,7 +48,7 @@ def subscribe(&block)
         end
       end # @watch.subscribe
       # when watch.subscribe ends - its because we got quit
-      _sincedb_write
+      @sincedb.write("shutting down")
```

Comment:
this api is a bit strange, it seems it's writing "shutting down" to the since db, why not do something like `@sincedb.flush_to_disk(:reason => "shutting down")`